### PR TITLE
Wide layout for endpoints

### PR DESCRIPTION
**Pull request by davifantasia**
Last PR to

* Close https://github.com/commercetools/commercetools-docs-kit/issues/445
* Fix https://github.com/commercetools/commercetools-docs-kit/issues/550
* Replace https://github.com/commercetools/commercetools-docs-kit/pull/594

### CODE REVIEW

**Comment by emmenko** [comment]
Shouldn't we additionally take VRT snapshots with the wider viewport?

**Comment by davifantasia** [comment]
> Shouldn't we additionally take VRT snapshots with the wider viewport?

Yes, that would make sense. I'll add that.

File: packages/gatsby-theme-docs/src/components/side-by-side.js [hunk 1]
```diff
@@ -3,38 +3,28 @@ import PropTypes from 'prop-types';
...
  * Allows the first and second child of this container to float side by side in large desktop sizes.
- *
- * NOTE about the `!important` override.
- * The <SideBySide> content is sometimes an entry in the UI Kit stack (e.g. in API docs).
- * The UI kit stack forces the top spacing and uses `!important`.
- * However, given that the SideBySide is responsive, the second item can sometimes be
- * to the right of the first and sometimes under it.
- * Therefore, we need different spacing "logic" here - the UI Kit stacks and inlines are
- * not intended for responsive content. As a consequence, without this override, the right
- * item would not be top aligned with the left item.
```
Comment (emmenko) [inline packages/gatsby-theme-docs/src/components/side-by-side.js:18]: Any reason for removing this comment?

Comment (davifantasia) [reply packages/gatsby-theme-docs/src/components/side-by-side.js:18]: > Any reason for removing this comment?

Nikolaus wrote it and removed it in his PR. See my comment here - https://github.com/commercetools/commercetools-docs-kit/pull/594/files#r475474449

**Comment by emmenko** [comment]
image

The "floatExample" container looks odd (the width). Is that how it's supposed to be?

**Comment by davifantasia** [comment]
> The "floatExample" container looks odd (the width). Is that how it's supposed to be?

Reducing the width was the only way I could find to prevent the  weird overflow you identified here https://github.com/commercetools/commercetools-docs-kit/pull/620#issuecomment-678162422

**Comment by emmenko** [comment]
Ok, just wanted to double check. We will for sure iterate on this as we go on.

**Review by emmenko — APPROVED** [review]

**Pull request merged**